style/topbar.css

```diff
@@ -0,0 +1,45 @@
+.topbar {
+  padding: 0px 30px;
+  display: flex;
+  flex-wrap: wrap;
+}
+.card-category {
+  height: 150px;
+  width: 250px;
+  margin: 50px 20px;
+  background-image: url("../assets/men.jpeg");
+  background-repeat: no-repeat;
+  background-position: center;
+  position: relative;
+  background-size: 100%;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.card-category h2 {
+  background: var(--white-color);
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  width: 100%;
+  transition: 0.3s;
+}
+.card-category:hover h2{
+    color: black;
+}
+@media (max-width: 767px) {
+  .topbar {
+    padding: 0px 0px;
```

Comment:
You can just write `padding: 0; `  it will do the same. And you don't need to give units when you are making it 0.